packages/devtools_app/lib/src/standalone_ui/vs_code/devtools.dart

```diff
@@ -84,6 +88,94 @@ class DevToolsSidebarOptions extends StatelessWidget {
   }
 }
 
+class _DevToolsExtensions extends StatefulWidget {
+  const _DevToolsExtensions({required this.api});
+
+  final VsCodeApi api;
+
+  @override
+  State<_DevToolsExtensions> createState() => _DevToolsExtensionsState();
+}
+
+class _DevToolsExtensionsState extends State<_DevToolsExtensions> {
+  ExtensionService? _extensionService;
+
+  @override
+  void initState() {
+    super.initState();
+    _initExtensions();
+  }
+
+  void _initExtensions() {
+    _extensionService = ExtensionService(ignoreServiceConnection: true);
+    unawaited(_extensionService!.initialize());
+  }
+
+  @override
+  void dispose() {
+    _extensionService?.dispose();
+    _extensionService = null;
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          'DevTools Extensions',
+          style: theme.textTheme.titleMedium,
+        ),
+        ValueListenableBuilder(
+          valueListenable: _extensionService!.visibleExtensions,
+          builder: (context, extensions, _) {
+            if (extensions.isEmpty) {
+              return const Padding(
+                padding: EdgeInsets.only(left: borderPadding),
+                child: Text('No extensions detected.'),
```

Comment:
Is it worth just hiding the section if there are none? (it'll probably be empty for most people, at least for a while, so this might just be extra noise)

Reply:
I was wondering if having a section here might help with discoverability. We could event have a help link that opens the DevTools extensions docs. Eventually, it would be nice if the devtools extensions docs could link to a pub.dev link that queries all packages that provide extensions (https://github.com/dart-lang/pub-dev/issues/7216).

@anderdobo @jacob314 do you have an opinion here on what we should show in the sidebar WRT to extensions? Should we hide the DevTools Extensions section when there are none available, or leave the section with an empty message "No extensions detected" for discoverability of the feature?

Reply:
For a cleaner UI, I think we should hide the DevTools Extensions section when there are none available.